src/board/system76/common/kbscan.c

```diff
@@ -7,8 +7,32 @@
 #include <board/keymap.h>
 #include <board/pmc.h>
 #include <board/power.h>
+#include <common/config.h>
 #include <common/debug.h>
 
+config_t kbscan_cfg_fnlock = {
+    .config_short = "Fn Lock",
+    .config_desc = "When enabled, F1-F12 will activate their default function",
+    .value = {
+        .min_value = 0,
+        .max_value = 1,
+        .value = 0 /* Default to disable */
+    },
+    .set_callback = NULL,
+};
+
+config_t kbscan_cfg_debounce = {
+    .config_short = "Keyboard Debounce Time",
```

Comment:
This isn't something I would expect to see configurable (I don't expect it to ever change). What value does it provide to be user configurable instead of defined by the firmware?

Reply:
I agree that it really isn't useful for most (any?) users to configure the debounce time and can be removed before anything is actually merged.

I'll go ahead and remove it when I next push up changes.

Reply:
This is now removed.